src/main/java/net/glowstone/entity/GlowLivingEntity.java

```
@@ -1243,6 +1224,24 @@ public boolean isHandRaised() {
         return false;
     }
 
+    @Override
+    public boolean isSwimming() {
+        // TODO: 1.13
+        throw new UnsupportedOperationException("Not implemented yet.");
```

Add `@Getter @Setter private boolean swimming` as a starting point, and move the TODO there.

---

I'd rather have this error thrown if a plugin uses unimplemented APIs than failing silently. At least then, incompatibility is made clear to the end user.